src/main/java/com/hcl/appscan/jenkins/plugin/scanners/StaticAnalyzer.java

```diff
@@ -192,7 +196,9 @@ public void validateSettings(JenkinsAuthenticationProvider authProvider, Map<Str
                     progress.setStatus(new Message(Message.WARNING, Messages.warning_include_sca_AppScan360()));
                     properties.remove(CoreConstants.INCLUDE_SCA);
                 }
-            } else if(properties.containsKey(CoreConstants.INCLUDE_SCA) && !ServiceUtil.hasScaEntitlement(authProvider)) {
+            }
```

Comment:
Is there a reason this was made an "if" as opposed to the previous "else if"? I think we want it to be "else if" because with this change, an AppScan 360 user will get 2 warning about SCA.

Reply:
I removed the else-if statement because the OSO & include-SCA will not be a valid case in Jenkins due to that users will not get 2 warnings.